src/Entity/Webhook.php

```diff
@@ -36,9 +47,9 @@ class Webhook
     /**
      * @var string
      */
+    #[ApiProperty(openapiContext: ['type' => 'string', 'enum' => ['delivery.assigned', 'delivery.started', 'delivery.failed', 'delivery.picked', 'delivery.in_transit', 'delivery.completed', 'order.created']])]
```

Comment:
why not keep using `'enum' => Webhook::EVENTS`?

Reply:
It was migrated automatically via a command, I think it resolved the actual value.